catalog/spatial/geowebcache/geowebcache-admin-plugin/src/main/webapp/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ -->
+<html class="no-js">
+
+<head>
+    <meta charset="utf-8">
+
+    <title>GeoWebCache Layers</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="">
+    <meta name="author" content="">
+
+    <link href="lib/bootswatch/flatly/bootstrap.min.css" rel="stylesheet">
+    <link href="lib/components-font-awesome/css/font-awesome.min.css" rel="stylesheet">
+
+    <link href="css/index.css" rel="stylesheet">
+
+    <!-- HTML5 shim, for IE6-8 support of HTML5 elements -->
+    <!--[if lt IE 9]>
+    <script src="lib/html5shiv/dist/html5shiv.js"></script>
```

Comment:
GitHub shows this as commented out, but I think this script is needed (boilerplate for admin plugins).  https://github.com/codice/ddf/blob/master/catalog/admin/module/catalog-admin-module-sources/src/main/webapp/index.html